extensions/reactive-messaging-http/runtime/src/main/java/io/quarkus/reactivemessaging/http/config/ReactiveHttpConfig.java

```diff
@@ -0,0 +1,102 @@
+package io.quarkus.reactivemessaging.http.config;
+
+import static java.util.regex.Pattern.quote;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Singleton;
+
+import org.eclipse.microprofile.config.Config;
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.eclipse.microprofile.config.spi.ConfigProviderResolver;
+
+import io.quarkus.reactivemessaging.http.QuarkusHttpConnector;
+import io.quarkus.reactivemessaging.http.QuarkusWebsocketConnector;
+
+/**
+ * ms@author Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
+ * <br>
+ * Date: 27/09/2019
+ */
+@Singleton
+public class ReactiveHttpConfig {
+    private static final String CONNECTOR = ".connector";
```

Comment:
All the non API classes should be in a runtime subpackage per convention.

Reply:
should be okay now